app/core/settings.py

```diff
@@ -23,15 +23,14 @@ class QueueMode(str, Enum):
 
 class StorageMode(str, Enum):
     LOCAL_STORAGE = "local_storage"
-    MEMORY = "memory"
     MONGO = "mongo"
 
 
 # pylint:disable=too-few-public-methods
 class Settings(BaseSettings):
     API_MODE: ApiMode = ApiMode.MEDIATOR
     QUEUE_MODE: QueueMode = QueueMode.LOCAL
-    STORAGE_MODE: StorageMode = StorageMode.MEMORY
+    STORAGE_MODE: StorageMode = StorageMode.LOCAL_STORAGE
```

Comment:
As discussed, removing the memory storage mode that doesn't work and defaulting to local storage